tutorials/t2-random-xp.md

```diff
@@ -0,0 +1,110 @@
+
+In the previous tutorial we had set up the `XpManager`, and now it's time to use it! We are going to do one of the most notorious things an xp bot does: giving users a random amount of xp on every message, so let's get started! Our file should look something around this lines:
+```javascript
+const Discord = new Discord.Client();
+const client = new Discord.Client();
+const DiscordXp = require("discord-xp");
+client.xp = new DiscordXp.XpManager("mongodb", {
```

Comment:
@MrAugu this could be an issue for TypeScript users since you can't add a property to an existing type.

Reply:
I see what you are talking about, but since DiscordXp starting `v2.0` is instance based, so that would either force users to spawn multiple instances which could be dangerous with local providers such as `JSON` or have users do crazy exports and imports to get module from here to there.

Reply:
Yeah, it definitely should be clearly documented, that's my point :)